mesonbuild/interpreterbase/__init__.py

```diff
@@ -87,7 +86,6 @@
     noPosargs,
     noKwargs,
     noArgsFlattening,
-    noSecondLevelHolderResolving,
```

Comment:
This doesn't look at all right (unless you planned to remove the decorator entirely from decorators.py, that is ;))

Reply:
Good catch, that was leftover from previous iteration. Added that line back.